summoner-tui/src/Summoner/Tui.hs

```diff
@@ -120,15 +120,22 @@ appNew dirs = App
                 withForm ev s (validateForm . ctrlD)
 
             -- Handle active/inactive checkboxes
-            VtyEvent (V.EvKey (V.KChar ' ') []) -> case getCurrentFocus s of
-                Nothing    -> withFormDef ev s
-                Just field -> handleCheckboxActivation ev s field
+            VtyEvent (V.EvKey (V.KChar ' ') []) ->
```

Comment:
I don't think this even should change. Packages in Haskell can't contain whitespaces, so we can just not to do anything at all with the prelude module in this case

Reply:
Okay. So we want to trigger this check every time a character in `[a..z] or [A..Z]` is entered.

Reply:
@Naman-sopho We want to trigger this event on `[a..z], [A..Z], [0..9], -`. This are the characters allowed in the package name. And then the module name field should be automatically filled using the value of prelude package name